submodules/myOffice/myOffice.js

```diff
@@ -1077,11 +1077,13 @@ define(function(require) {
 			if (self.isActive()) {
 				// First we check if the user hasn't seen the walkthrough already
 				// if he hasn't we show the walkthrough, and once they're done with it, we update their user doc so they won't see the walkthrough again
-				self.myOfficeHasWalkthrough(function() {
-					self.myOfficeShowWalkthrough(function() {
-						self.myOfficeUpdateWalkthroughFlagUser();
+				if (!self.appFlags.disableShowfirstUseWalkthrough) {
```

Comment:
Same here, it would be better to use the guard pattern to exit the function if the condition is not met:
```suggestion
				if (self.appFlags.disableFirstUseWalkthrough) {
					return;
				}
```